ouroboros-network-framework/test/Test/Ouroboros/Network/Server2.hs

```diff
@@ -2504,6 +2592,40 @@ prop_inbound_governor_no_invalid_traces serverAcc (ArbDataFlow dataFlow)
                        (Script (toBearerInfo absBi :| [noAttenuation]))
                        maxAcceptedConnectionsLimit events
 
+-- | Property wrapping `multinodeExperiment`.
+--
+-- Note: this test coverage of inbound governor state transitions.
+-- TODO: Fix transitions that are not covered. #3516
+prop_inbound_governor_transitions_coverage :: Int
+                                           -> ArbDataFlow
+                                           -> AbsBearerInfo
+                                           -> MultiNodeScript Int TestAddr
+                                           -> Property
+prop_inbound_governor_transitions_coverage serverAcc (ArbDataFlow dataFlow)
+                                           absBi
+                                           (MultiNodeScript
+                                                events) =
+  let trace = runSimTrace sim
+
+      remoteTransitionTraceEvents :: [RemoteTransitionTrace SimAddr]
+      remoteTransitionTraceEvents = withNameTraceEvents trace
+
+      transitionsSeen = nub [ tran
+                            | TransitionTrace _ tran
+                                <- remoteTransitionTraceEvents]
+      transitionsSeenNames = map (snd . validRemoteTransitionMap)
+                                 transitionsSeen
+
+   in coverTable "valid transitions"
+                  [ (n, 0.01) | n <- allValidRemoteTransitionsNames ] $
+      tabulate   "valid transitions" transitionsSeenNames
+      True
```

Comment:
The same here, does it passes if we plug `checkCoverage`?

Reply:
Same as above